app/views/checkins/_player_show.html.haml

```diff
@@ -7,4 +7,4 @@
 = render 'checkins/current_puzzle_info', locals: {puzzle: @checkin.next_puzzle, location_coordinates: @location_coordinates, next_location_coordinates: @next_location_coordinates}
 
 
-%p Details on your next puzzle have also been sent to your email.
+%p Details on your next puzzle may have also been sent to your email.
```

Comment:
Why the ambiguity? Did email delivery break in the app, or was it never set up for this deployment?